### PR TITLE
Maker: error overlay changes

### PR DESCRIPTION
**Pull request by islemaster**
Adds two new controls to the Maker error overlay.  We've been getting complaints today from App Lab users who accidentally ended up with Maker Toolkit enabled, and are seeing the error overlay every time they run their program (because they don't have a board) - we're hoping these links will help a little bit.  There's also another change on the way that will add a confirmation dialog when enabling Maker, to help reduce the number of people that accidentally end up in this state.

image

The "Get Help" link opens the [Maker Setup Page](https://studio.code.org/maker/setup) in a new tab.  The "Disable Maker Toolkit" link saves the project with maker disabled and reloads the page.

I also added a storybook story for the Maker overlays to help with iterating on these changes:

image

### CODE REVIEW

File: apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx [hunk 1]
```diff
@@ -19,27 +21,44 @@ const overlayDimensionsPropTypes = {
...
     scale: getVisualizationScale(state),
     isConnecting: isConnecting(state),
     hasConnectionError: hasConnectionError(state),
+  }),
+  () => ({
+    handleDisableMaker: () => project.toggleMakerEnabled(),
+    handleTryAgain: () => {
+      studioApp.resetButtonClick();
+      studioApp.runButtonClick();
+    },
```
Comment (Bjvanminnen) [inline apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx]: This feels like a bit of an abuse of connect. Connect is meant to match props to dispatches. In neither of these are we dispatching actions. Couldn't these just be methods on the component (or passed in by the caller)?

File: apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx [hunk 2]
```diff
@@ -64,42 +83,40 @@ const style = {
     padding: '2em',
     textAlign: 'center',
   },
+  footer: {
+    flex: '0 0 0',
```
Comment (Bjvanminnen) [inline apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx]: I believe IE11 only has partial support for flexbox. Might be worth a look in that browser.

File: apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx (hunk 1 above)
Comment (islemaster) [reply apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx]: I'll have the caller pass these in.

File: apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx (hunk 2 above)
Comment (islemaster) [reply apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx]: You're right - it was having issues, but changing to always use `flex-basis: auto` (which here looks like `flex: '0 0 auto',`) fixed things up.

Here it is working in IE11:
image

**Pull request merged**